src/visualization/types/SimpleTable/view.tsx

```diff
@@ -17,14 +18,18 @@ interface Props extends VisualizationProps {
 }
 
 const SimpleTable: FC<Props> = ({properties, result}) => {
-  return (
-    <div className="visualization--simple-table" data-testid="simple-table">
-      <PaginationProvider total={result?.table?.length || 0}>
-        <PagedTable properties={properties} result={result} />
-        <PageControl />
-      </PaginationProvider>
-    </div>
-  )
+  const parsed = parseFromFluxResults(result)
```

Comment:
you need to revert these changes, and wrap them behind a feature flag. the way you have the flag set up currently is that you're preventing the user from using simpletable, but still making all existing visualizations of this type fo through the new logic. we want to gate the code change, not the user experience for this visualization.

Reply:
Good point!! Thank you for noticing that, that was a blunder on my part, I get it now. I'll fix that asap.

Reply:
this declaration also needs to be put inside of the flag or else we're double parsing (an expensive operation) the results for every simpletable

Reply:
Ah! Got it, will fix that 👍

Reply:
Just fixed it.